riotfile.py

```diff
@@ -236,6 +236,12 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     # Test with the latest version of Python only
                     pys=MAX_PYTHON_VERSION,
                 ),
+                Venv(
+                    name="tracer-legacy-atrrs",
+                    pkgs={"cattrs": "<23.2.0", "attrs": "==20.1.0"},
+                    # Test with the latest version of Python only
```

Comment:
this says test with latest version of Python, but we are using `MIN_PYTHON_VERSION` ?

Reply:
Testing all python versions with the legacy attrs version seemed like over kill.

Since 3.12 does not support attrs v20.1, I used min version. 

I guess we could've used 3.11 instead.